expyfun/tests/test_experiment_controller.py

```diff
@@ -620,6 +620,64 @@ def test_sound_card_triggering(hide_window):
         ec.load_buffer([1e-2])
         ec.start_stimulus()
         ec.stop()
+    # Test the drift triggers
+    with pytest.warns(UserWarning, match='Drift triggers overlap with onset '
+                      'triggers'):
```

Comment:
These context managers should be as targeted as possible, so should cover just the `load_buffer` step I think

Reply:
Makes sense, will update.

Reply:
Also removed WIP from the title, since I think we're pretty much good to go on our end.